src/Utils/SchemaExtender.php

```diff
@@ -609,11 +609,7 @@ static function (string $typeName) use ($schema) {
             }
         }
 
-        $schemaExtensionASTNodes = count($schemaExtensions) > 0
-            ? ($schema->extensionASTNodes
-                ? array_merge($schema->extensionASTNodes, $schemaExtensions)
-                : $schemaExtensions)
-            : $schema->extensionASTNodes;
+        $schemaExtensionASTNodes = array_merge($schema->extensionASTNodes, $schemaExtensions);
```

Comment:
both are always arrays so the conditions are useless, we can just merge them... empty arrays don't matter

Reply:
Actually, it looks like they are not. Check travis:

```
1) GraphQL\Tests\Type\ValidationTest::testRejectsASchemaExtendedWithInvalidRootTypes
array_merge(): Expected parameter 1 to be an array, null given
/home/travis/build/webonyx/graphql-php/src/Utils/SchemaExtender.php:612
/home/travis/build/webonyx/graphql-php/tests/Type/ValidationTest.php:530
```

Reply:
@spawnia yeah I saw it, just didn't have the time to look into it yet.

Reply:
@spawnia Should be fixed now.